selftest/tracing-by-offset/main.go

```diff
@@ -9,14 +9,14 @@ import (
 	"syscall"
 
 	bpf "github.com/aquasecurity/libbpfgo"
-	"github.com/aquasecurity/libbpfgo/helpers"
 	"github.com/aquasecurity/libbpfgo/selftest/common"
+	"github.com/aquasecurity/tracee/pkg/symbols"
 )
 
 func main() {
 	funcName := fmt.Sprintf("__%s_sys_mmap", common.KSymArch())
 
-	kst, err := helpers.NewKernelSymbolTable()
+	kst, err := symbols.NewKernelSymbolTable(false, false)
```

Comment:
@yanivagman as you can see, as symbols pkg is not under common, it demands a full import of Tracee, changing the [go version and toolchain](https://github.com/aquasecurity/libbpfgo/pull/473/files#diff-49b92141949f2a52b8dd64f7e611f239a7145f3b0cf165b731f8797050d81fe1R3-R5). This is not a problem for these two selftests. JFYI.

Reply:
You're right, we can get rid of that dependency completely.